spec.html

```diff
@@ -101,6 +101,9 @@ <h1>Runtime Semantics: Evaluation</h1>
               1. <ins>If Type(_value_) is not String, then</ins>
                 1. <ins>Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).</ins>
                 1. <ins>Return _promiseCapability_.[[Promise]].</ins>
+              1. <ins>If _key_ is *"type"* and _value_ is not an entry of !HostGetSupportedExtraModuleScriptTypes(), then</ins>
```

Comment:
```suggestion
              1. <ins>If _key_ is *"type"* and ! HostGetSupportedExtraModuleScriptTypes() does not contain _value_, then</ins>
```
(lists have elements, not entries)